src/test/java/com/ifactory/press/db/solr/highlight/SolrPostingsHighlighterTest.java

```diff
@@ -46,12 +48,17 @@ private void assertSnippet(String q, String expectedSnippet) throws SolrServerEx
         query.set("hl.tag.ellipsis", "¦");
         query.set("f.text.hl.snippets", 3); // override value of 3 specified in solrconfig.xml
         // query.set("hl.maxAnalyzedChars", 250000);
-        QueryResponse resp = solr.query(query);
+        QueryResponse resp = null;
+        try {
+            resp = solr.query(query);
+        } catch (IOException ex) {
+            Logger.getLogger(SolrPostingsHighlighterTest.class.getName()).log(Level.SEVERE, null, ex);
+        }
```

Comment:
This question is more about my ignorance of Java and Junit. Does catching this exception here and logging, but _not_ re-raising affect the performance of the test at all? E.g. does the test just continue after the `log()` call?